modules/global/src/com/haulmont/cuba/core/global/MetadataTools.java

```diff
@@ -706,6 +706,22 @@ public String getDatabaseColumn(MetaProperty metaProperty) {
     }
 
     /**
+     * Determine whether the given metaProperty relates to at least one non local property
+     */
+    public boolean isRelatedToNonLocalProperty(MetaProperty metaProperty) {
```

Comment:
Move it to EntityInspectorEditor, as the method looks trivial and rarely needed.